lib/blob.js

```diff
@@ -420,13 +420,21 @@ Blob.prototype.request = function request(method, path, query, headers, data) {
         // Parse error message
         var data = xml.parseError(res);
 
+        var resMSHeaders = {};
+        Object.keys(res.headers).forEach(h => {
+          if (h.startsWith('x-ms-')) {
```

Comment:
we're sure none of these headers should be avoided logging?

Reply:
Not sure at all -- we have no idea what they are!

Reply:
A look through the docs suggests it's versions and stuff.  The hope is to catch the headers [here](https://docs.microsoft.com/en-us/azure/virtual-machines/troubleshooting/troubleshooting-throttling-errors), but it's not clear that the storage API produces them..